$OpenBSD: patch-include_mupdf_fitz_context_h,v 1.1 2017/08/14 09:59:07 pirofti Exp $

Fix DEBUG builds for programs that include this header.
See https://bugs.ghostscript.com/show_bug.cgi?id=698353.

Index: include/mupdf/fitz/context.h
--- include/mupdf/fitz/context.h.orig
+++ include/mupdf/fitz/context.h
@@ -570,7 +570,7 @@ extern fz_alloc_context fz_alloc_default;
 /* Default locks */
 extern fz_locks_context fz_locks_default;
 
-#if defined(MEMENTO) || !defined(NDEBUG)
+#if defined(MEMENTO)
 #define FITZ_DEBUG_LOCKING
 #endif
 
